### PR TITLE
RDF: Import dc:description as Extra

### PR DESCRIPTION
**Pull request by adam3smith**
Closes https://github.com/zotero/translators/issues/1456
Fixes Zotero RDF roundtriping. 
Also remove some whitespace

### CODE REVIEW

**Comment by adam3smith** [comment]
@dstillman if you could take a very quick look at https://github.com/zotero/translators/pull/1461/files?w=1 to make sure there isn't something fundamentally objectional at the approach I'm taking, that'd be super helpful.

**Comment by dstillman** [comment]
I'll look at the rest, but can you remove whitespace changes to start? (Shouldn't be in the same commit, and indenting empty lines is our standard code style anyway. We'll try to get an eslint setup everywhere to help with things like this.)

**Comment by adam3smith** [comment]
Done

**Comment by zuphilip** [comment]
(
> We'll try to get an eslint setup everywhere to help with things like this.

@dstillman The script here https://github.com/zotero/translators/blob/master/.ci/checkTranslator.sh#L108-L119 can produce some warnings w.r.t. JSHint, but we skip them in Travis CI currently here in translators, because most of the (old) translator would not pass that. And maybe the rules would need to be adjusted.
)

File: RDF.js [hunk 1]
```diff
@@ -965,6 +969,12 @@ function importItem(newItem, node) {
 	// abstract
 	newItem.abstractNote = getFirstResults(node, [n.eprints+"abstract", n.prism+"teaser", n.prism2_0+"teaser", n.prism2_1+"teaser", n.og+"description",
 		n.bibo+"abstract", n.dcterms+"abstract", n.dc+"description.abstract", n.dcterms+"description.abstract", n.dc1_0+"description"], true);
+
+	//extra for Zotero RDF
```
Comment (zuphilip) [inline RDF.js]: Add one space after `//`.

File: RDF.js [hunk 2]
```diff
@@ -1044,7 +1054,7 @@ function importItem(newItem, node) {
 	// description/attachment note
 	if(newItem.itemType == "attachment") {
 		newItem.note = getFirstResults(node, [n.dc+"description", n.dc1_0+"description", n.dcterms+"description"], true);
-	} else if (!newItem.abstractNote) {
+	} else if (!newItem.abstractNote && !isZoteroRDF) {
 		newItem.abstractNote = getFirstResults(node, [n.dc+"description", n.dcterms+"description"], true);
 	}
```
Comment (zuphilip) [inline RDF.js]: Can we move the lines from above here, such that all cases for `dc:description` are handled in one place? Something like (not tested):
```js
 	// dc:description
 	if (newItem.itemType == "attachment") {
 		newItem.note = getFirstResults(node, [n.dc+"description", n.dc1_0+"description", n.dcterms+"description"], true);
 	} else if (isZoteroRDF) {
		// for ZoteroRDF dc:description holds the data from extra
		newItem.extra = newItem.extra ? newItem.extra + getFirstResults(node, [n.dc+"description"], true) : getFirstResults(node, [n.dc+"description"], true);
	} else if (!newItem.abstractNote) {
 		newItem.abstractNote = getFirstResults(node, [n.dc+"description", n.dcterms+"description"], true);
 	}
```
Maybe even test it directly here without an additional variable.

**Comment by zuphilip** [comment]
@dstillman Any additional comments from your side?

**Comment by adam3smith** [comment]
@dstillman bump -- only a couple of lines to check over, should be quick.

**Comment by zuphilip** [comment]
@dstillman I will merge this the next days. Please let me know soon when you don't agree...

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**